flatdata-rs/lib/src/generator.rs

```diff
@@ -128,6 +142,11 @@ pub enum GeneratorError {
 impl std::fmt::Display for GeneratorError {
     fn fmt(self: &Self, f: &mut std::fmt::Formatter) -> Result<(), std::fmt::Error> {
         match self {
+            GeneratorError::NotFound {path }=> {
+                writeln!(f, "{} could not be executed:\n", path.display())?;
```

Comment:
Unfortunately, display for error should produce a short single line error description. I don't think we can add a lot of context information here.

Reply:
I'm curious, why?